modules/datetime.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+import { DateTime } from "./luxon.js";
+
```

Comment:
- [ ] It would be nice if you could fix these linter checks so your code is professional and standard. Disabling the linter checks defies its purpose and it's there to help make you write a clean, modern, professional, and error-free codebase. Kindly fix them 👍